projects/MarlQ/proposal/README.md

```diff
@@ -0,0 +1,56 @@
+I imagine others would be interested in this feature as well. 
+
+## Proposal
+
+The proposed feature is a simple, momentary simulation of the volume displacement of a soft object caused by one (or more) solid object(s).
+This displacement would directly impact the vertex positions of the soft object, changing the mesh.
+Ideally, these deformations could then be saved as shape keys.
+Physical properties like elasticity or material density of the soft object could be factored in as parameters, to control the range of the volume displacement and its strength.
+However, other typical physical properties like gravity, or even push resistance should not be considered.
+The former (gravity) would create undesirable side effects. 
+The latter (push resistance) does not make sense, considering it is a momentary simulation --- this means that the hard object already intersects with the soft object which would indicate a possibly infinite force being exerted onto the soft object, rendering any push resistance meaningless.
+
+Unfortunately, I could not yet find much regarding this topic, except a video showcasing a simple method utilizing booleans for Houdini, which labels the method as 'boolean denting' (Boolean Denting (VDB Basics)).
+The results from that method look alright for some cases, but the resulting dents are way too exaggerated, making them look more artistic rather than realistic.
+Jasper Flick also made a tutorial for a similar effect for the Unity engine, mentioning the use of the inverse-square law in conjunction with the force exerted onto the soft object (Mesh Deformation - Making a Stress Ball).
+
+## Implementation
+As a Blender addon, the project will be implemented in Python.
+For now, the connected functions will be implemented one after the other, starting with a simple displacement operation, which displaces all overlapping vertices of a soft object with another, hard object.
+To enable better shadow and light interactions, here, already, can be made improvements by creating a small gap betweent the objects.
+
+Then, the remaining displaced volume will distributed across the rest of the object.
+How exactly it will be distributed remains an open question, but there multiple posibilities (e.g. simple linear falloff, inverse-square).
+Material properties could change the exact way the force caused by the volume displacement is distributed.
+Considerations will have to be made how to handle the overlapping points (when the objects intersect at multiple areas).
+
+Lastly, the dents will be smoothed, based on the materials shear resistance, as the dents usually do not form in immediate vicinity of the displacing object.
+
+## References
+
+- Entagma. Boolean Denting (VDB Basics). Dec 26, 2019.  https://www.youtube.com/watch?v=9VcIP7ByDGE
+- Jasper Flick. Mesh Deformation - Making a Stress Ball (Unity C# Tutorials). https://catlikecoding.com/unity/tutorials/mesh-deformation/
```

Comment:
The references are helpful to demonstrate the ideal but not yet enough to guide the implementation procedural.